packages/extension-link/src/helpers/clickHandler.ts

```diff
@@ -11,7 +11,7 @@ export function clickHandler(options: ClickHandlerOptions): Plugin {
     key: new PluginKey('handleClickLink'),
     props: {
       handleClick: (view, pos, event) => {
-        if (event.button !== 1) {
+        if (event.button !== 0) {
```

Comment:
Please add more information to your pull request.

From your change I'm not sure what exactly it's doing.

I think not being able to click a Link is expected behaviour when you're in "edit" mode. I think when you made the editor read-only you can click links.

I think rather than completely hard coding this we could add an option to the extension to allow devs to enable link clicking.

How about adding an `allowClick` option to the link extension which then allows the click handler to handle left mouse clicks?

Reply:
@bdbch I guess it is related to https://github.com/ueberdosis/tiptap/issues/3915
Confirm it's not working anymore (before PR), see https://codesandbox.io/s/focused-gagarin-hun49u?file=/src/App.js.

Reply:
@bdbch are you sure? What about existing `openOnClick`: https://tiptap.dev/api/marks/link#open-on-click

Reply:
Seems to be good